src/Special/PSSpecialShare.php

```diff
@@ -295,4 +302,34 @@ public function doShare( string $usr, PSShare $share, PSRender $render ) {
 		}
 		return false;
 	}
+
+	/**
+	 * @param string $string
+	 * @param string $start
+	 * @param string $end
+	 *
+	 * @return false|string
+	 */
+	private static function getStringBetween( string $string, string $start, string $end ) {
```

Comment:
Like with the other pull request, please change this to 
```
private static function getStringBetween( string $string, string $start, string $end ) {
 $startPosInString = strpos( $string, $start );
 if ( $startPosInString === false ) {
  // Do something, like throw an exception or return empty string or full string, I don't know.
 }
 $startPosInString += strlen( $start );
 $endPosInString = strrpos( $string, $end, $startPosInString );
 if ( $endPosInString === false ) {
  // Do something, like throw an exception or return empty string or full string, I don't know.
 }
 $length = $endPosInString - $startPosInString;
 return substr( $string, $startPosInString, $length );
}
```
Or something very similar